src/app/root-content/dashboard/widgets/widgets.component.scss

```diff
@@ -0,0 +1,77 @@
+:host {
+	display: block;
+	.ui.cards .ui.card.widget-container {
+		box-shadow: none;
+	}
+
+	.ui.fluid.card {
+		font-size: .95em;
+		box-shadow: none;
+	}
+
+	.ui.card.error {
+		/deep/ .ui.table, /deep/ .ui.header {
+			color: gray;
+		}
+		/deep/ .widget-text {
+			flex: auto;
+			color: gray;
+
+		}
+
+		/deep/ .widget-header {
+			background-color: #bcbec0;
+			padding: .5em .5em;
+		}
+	}
+
+	.ui.cards {
+		.ui.card {
+			font-size: .95em;
+			width: 245px;
+
+			/deep/ .widget-header {
+				background-color: orange;
+				padding: .5em .5em;
+			}
+
+			/deep/ .widget-button {
+				cursor: pointer;
+			}
+
+			/deep/ .widget-body {
+				display: flex;
+				flex-direction: row;
+			}
+
+			/deep/ .widget-icon {
+				height: 60px;
+			}
+
+			/deep/ .widget-text {
+				flex: auto;
+				color: black;
+				line-height: normal;
+				font-size: 50px;
+			}
+		}
+
+		.ui.card.error {
+			/deep/ .widget-text {
+				flex: auto;
+				color: gray;
+				line-height: normal;
+				font-size: 50px;
+			}
+			/deep/ .widget-header {
+				background-color: #bcbec0;
+				padding: .5em .5em;
+			}
+		}
+	}
+
+	/deep/ .ui.card.hidden {
+		display: none;
+	}
+
+}
```

Comment:
Some rules must go inside the widget itself and not in the container.
Remember to apply rules that must make the component to work by itself, let's say whenever you hove the title, it collapses. Hence that rule must go inside the widget SCSS file instead of the container.
The container can apply customization rules that don't change initial behavior, like color or font-sizes, otherwise what would be the use of SCSS file of the widget components if you define everything in the container file?